### DISCUSSION TITLE
getTextareaProps is missing ref just like getInputProps

### DISCUSSION THREAD

**Discussion by punkpeye** [edmundhung/conform#714] [Ideas]
This is how I use `getInputProps`.

```ts
const messageInputProps = getInputProps(fields.message, {
  ref: messageInputControl.register,
  type: 'text',
});
```

I try adopting `getTextareaProps`, but was blocked because it does not support `ref`.

**Comment by edmundhung** [comment]
`getInputProps` do not accept any `ref` nor return any `ref`. If you want all the properties in one object, feel free to spread it over with a extra ref like this:

```tsx
const messageInputProps = {
  ...getInputProps(fields.message, { type: 'text' }),
  ref: messageInputControl.register,
};
```
